gnomad_qc/v3/annotations/generate_freq_data.py

```diff
@@ -132,10 +134,25 @@ def main(args):
                 "Filtering MT columns to HGDP + 1KG samples that pass specialized sample QC"
             )
             hgdp_1kg_meta = hgdp_1kg_subset_annotations().ht()
+
+            # Note: Need to use sample names with the v3.1:: prefix
```

Comment:
```suggestion
            # Note: Sample IDs in MT have v3.1:: prefix, but sample IDs in hgdp tgp meta do not
            # Need to add prefix to hgdp tgp meta IDs to filter samples in MT correctly
```
this seems more descriptive (assuming I understood this part correctly)